Gems/ScriptCanvasDeveloper/Code/Editor/Source/Tools/TranslationBrowser/TranslationBrowser.cpp

```diff
@@ -0,0 +1,897 @@
+                        methodCategoryString.m_key = ScriptCanvasEditor::TranslationHelper::GetKey(ScriptCanvasEditor::TranslationContextGroup::ClassMethod, className.c_str(), methodPair.first.c_str(), ScriptCanvasEditor::TranslationItemType::Node, ScriptCanvasEditor::TranslationKeyId::Category);
+
+                        if (!methodCategoryString.GetDisplayString().empty())
+                        {
+                            methodEntry.m_details.m_category = methodCategoryString.GetDisplayString();
+                        }
+                        else
+                        {
+                            if (!MethodHasAttribute(behaviorMethod, AZ::ScriptCanvasAttributes::FloatingFunction))
+                            {
+                                methodEntry.m_details.m_category = details.m_category;
+                            }
+                            else if (MethodHasAttribute(behaviorMethod, AZ::Script::Attributes::Category))
+                            {
+                                methodEntry.m_details.m_category = GraphCanvasAttributeHelper::ReadStringAttribute(behaviorMethod->m_attributes, AZ::Script::Attributes::Category);;
+                            }
+
+                            if (methodEntry.m_details.m_category.empty())
+                            {
+                                methodEntry.m_details.m_category = "Other";
+                            }
+                        }
+
+                        // Arguments (Input Slots)
+                        if (behaviorMethod->GetNumArguments() > 0)
+                        {
+                            for (size_t argIndex = 0; argIndex < behaviorMethod->GetNumArguments(); ++argIndex)
+                            {
+                                // Generate the OLD style translation key
+                                AZStd::string oldClassName = className;
+                                AZStd::to_upper(oldClassName.begin(), oldClassName.end());
+                                AZStd::string oldMethodName = cleanName;
+                                AZStd::to_upper(oldMethodName.begin(), oldMethodName.end());
+                                AZStd::string oldKey = AZStd::string::format("%s_%s_PARAM%zu_NAME", oldClassName.c_str(), oldMethodName.c_str(), argIndex);
+                                AZStd::string oldTooltipKey = AZStd::string::format("%s_%s_PARAM%zu_TOOLTIP", oldClassName.c_str(), oldMethodName.c_str(), argIndex);
+
+                                const AZ::BehaviorParameter* parameter = behaviorMethod->GetArgument(argIndex);
+
+                                Argument argument;
+
+                                AZStd::string argumentKey = parameter->m_typeId.ToString<AZStd::string>();
+                                AZStd::string argumentName = parameter->m_name;
+                                AZStd::string argumentDescription = "";
```

Comment:
Default constructor defaults constructs the string as empty and it is ever so slightly faster